arxiv/marxdown/factory.py

```diff
@@ -59,14 +69,12 @@ def create_web_app(build_path: Optional[str] = None,
         # We build the blueprint on the fly, so that we get dynamic routing
         # to content pages.
         app.register_blueprint(
-            routes.get_blueprint(
-                build_path,
-                with_search=with_search
-            )
+            routes.get_blueprint(build_path, with_search=with_search)
         )
 
-    app.jinja_env.filters['format_datetime'] = format_datetime
-    app.jinja_env.filters['simepledate'] = simepledate
-    app.jinja_env.filters['pretty_path'] = pretty_path
+    app.jinja_env.filters['format_datetime'] = format_datetime  # pylint: disable=no-member
+    app.jinja_env.filters['simepledate'] = simepledate  # pylint: disable=no-member
```

Comment:
Did you mean:
```suggestion
    app.jinja_env.filters['simpledate'] = simpledate  # pylint: disable=no-member
```

Reply:
Aha! Looks like the typo was already there (I just added the pylint flag)... looks like the typo is everywhere, so I'll fix globally

Reply:
Fixed in 04209d6